storage/engine/rocksdb/db.cc

```diff
@@ -1179,10 +1160,14 @@ class BaseDeltaIterator : public rocksdb::Iterator {
     return merged_.data == NULL;
```

Comment:
`ProcessDeltaKey` advances the iterator until either the delta iterator is not valid or the key has changed. In either case, we've advanced past the last entry for the current key.